ncl/ncl_index/ncl-index-table.csv

```diff
@@ -1,3 +1,4 @@
 NCL Function, Description, Python Equivalent, Notes
 `days_in_month <https://www.ncl.ucar.edu/Document/Functions/Built-in/days_in_month.shtml>`__,"Calculates the number of days in a month given month and year","``cftime.datetime().daysinmonth``",`details <../ncl_entries/days_in_month.ipynb>`__
 `day_of_week <https://www.ncl.ucar.edu/Document/Functions/Built-in/day_of_week.shtml>`__,"Calculates the day of the week given month, day, and year","``cftime.datetime().dayofwk``",`details <../ncl_entries/day_of_week.ipynb>`__
+`spectral analysis <https://www.ncl.ucar.edu/Applications/spec.shtml>`__," Calculates spectra of a series.","``scipy.signal.periodogram()``, ``scipy.signal.csd()`, or ``scipy.signal.coherence()``",`example notebook <../ncl_entries/NCL_spectral_analysis.ipynb>`__
```

Comment:
Would it be useful to link to the specific functions [`specx_anal`](https://www.ncl.ucar.edu/Document/Functions/Built-in/specx_anal.shtml) and [`specxy_anal`](https://www.ncl.ucar.edu/Document/Functions/Built-in/specxy_anal.shtml) in the first column as well? In case people are searching for a specific function. @anissa111 if a notebook covers multiple functions, should each function get its own row (that links to the same notebook)? This is how I have trig functions set up

Reply:
Thanks addressed this in latest pushes.